composer.json

```diff
@@ -25,7 +25,7 @@
         "php": "^7.2",
         "ext-intl": "*",
         "dompdf/dompdf": "^0.8.0",
-        "illuminate/support": "^5.8|^6.0",
+        "illuminate/support": "^5.8|^6.0|^7.0",
```

Comment:
Thanks for the PR!

Can you change it to:

```
 "^6.0|^7.0|^8.0"
```

?

Reply:
Hi @sandervanhooft, 

No problem, might as well add Laravel 8.x compatibility while we are at it. I have added version illuminate/support ^8.0 to the composer.json and ran the unit tests again on a clean Laravel 8.0 installation. All tests passed.